libs/cdk/src/lib/utils/directives/line-clamp/line-clamp.directive.ts

```diff
@@ -181,6 +181,7 @@ export class LineClampDirective implements OnChanges, AfterViewInit, OnDestroy {
         this._originalText = event.fdLineClampTargetText;
         this.reset();
         this.refreshClamp();
+        this._checkLineCount();
```

Comment:
Good catch. `_checkLineCount()` already calls `reset()` and `refreshClamp()` so you can remove those above

Reply:
Got it. I removed the duplicated function calls and tested OK.
Would you check it please. Thanks!